src/common/options.js

```diff
@@ -132,7 +134,7 @@ function buildOptionsPage() {
           .append($('<input>', { type: 'checkbox', id: setting.name, name: setting.name, 'aria-describedby': setting.name + 'HelpBlock' }))
           .append($('<div>', { class: 'option-description' })
             .append($('<label>', { for: setting.name, text: setting.title }))
-            .append($('<span>', { id: setting.name + 'HelpBlock', class: 'help-block', text: setting.description }))))
+            .append($('<span>', { id: setting.name + 'HelpBlock', class: 'help-block', html: description }))))
```

Comment:
I can see exactly why this makes it easier for us, but unfortunately this would likely get us knocked back during Firefox review. They're a bit sensitive about potentially adding injection vulnerabilities should we ever switch to being able to load settings remotely.

Maybe we can use Markdown or similar so it goes through a safe processing step?